### ISSUE TITLE
Yarn install raises import declaration conflict 

### ISSUE THREAD

**Issue by hkache** [forcedotcom/dependencies-cli#5]
Plugin install is also failing silently for me:
<img width="969" alt="dependency" src="https://user-images.githubusercontent.com/2465122/72279336-35627400-3636-11ea-9d33-b2da750b98a5.png">

I tried to linked it localy after git cloning it, and got the following error:
`error dependencies-cli@2.0.1: The engine "yarn" is incompatible with this module. Expected version ">=1.15.2". Got "1.12.3"`
After updated yarn, I'm getting this error while running `yarn install`:
```
node_modules/@salesforce/core/lib/config/configStore.d.ts:4:10 - error TS2440: Import declaration conflicts with local declaration of 'ConfigContents'.

4 import { ConfigContents } from './configStore';
```
<img width="1080" alt="yarn" src="https://user-images.githubusercontent.com/2465122/72279348-3a272800-3636-11ea-8680-8f55c56fa4cd.png">

_Originally posted by @FabienTaillon in https://github.com/forcedotcom/dependencies-cli/issues/4#issuecomment-573788995_

**Comment by hkache** [comment]
@FabienTaillon try remove the conflict, i.e. `rm node_modules/@salesforce/core/lib/config/configStore.d.ts`. 

May need to remove `yarn.lock` before running `yarn install` again.

**Comment by hkache** [comment]
Fixed with version 2.0.1 by raising @salesforce/core and @salesforce/command versions.
